scanpy/readwrite.py

```diff
@@ -913,36 +913,26 @@ def _get_filename_from_key(key, ext=None) -> Path:
 
 
 def _download(url: str, path: Path):
-    try:
-        import ipywidgets
-        from tqdm.auto import tqdm
-    except ModuleNotFoundError:
-        from tqdm import tqdm
-    from urllib.request import urlopen, Request
-
-    blocksize = 1024 * 8
-    blocknum = 0
+    from tqdm.auto import tqdm
```

Comment:
Can you keep the check for `ipywidgets`? Using `tqdm.auto` in a notebook errors without it (seems it's not so `auto`)

Using tqdm 4.51.0:

```python
from tqdm.auto import tqdm
from time import sleep

for i in tqdm(range(100)):
    sleep(0.05)
```

```pytb
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-3-542c7d78a2ff> in <module>
----> 1 for i in tqdm(range(100)):
      2     sleep(0.05)

~/miniconda3/envs/tqdm_nowidgets/lib/python3.8/site-packages/tqdm/notebook.py in __init__(self, *args, **kwargs)
    221         unit_scale = 1 if self.unit_scale is True else self.unit_scale or 1
    222         total = self.total * unit_scale if self.total else self.total
--> 223         self.container = self.status_printer(
    224             self.fp, total, self.desc, self.ncols)
    225         self.sp = self.display

~/miniconda3/envs/tqdm_nowidgets/lib/python3.8/site-packages/tqdm/notebook.py in status_printer(_, total, desc, ncols)
     94         # Prepare IPython progress bar
     95         if IProgress is None:  # #187 #451 #558 #872
---> 96             raise ImportError(
     97                 "IProgress not found. Please update jupyter and ipywidgets."
     98                 " See https://ipywidgets.readthedocs.io/en/stable"

ImportError: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html
```

Reply:
Didn't know this caused an issue. I've reverted this.